pegasus/test/test_pegasus_documents.rb

```diff
@@ -23,7 +23,7 @@ def test_pegasus_documents
       "#{page[:site]}#{page[:uri]}"
     end
     CDO.log.info "Found #{documents.length} Pegasus documents."
-    assert_operator documents.length, :>, 1300
+    assert_operator documents.length, :>, 500
```

Comment:
This number is so much lower here because I also updated this [here](https://github.com/code-dot-org/code-dot-org/pull/60530) and have yet to merge that.